libcextract/SymbolExternalizer.hh

```diff
@@ -30,6 +31,21 @@ struct ExternalizerLogEntry
   ExternalizationType Type;
 };
 
+struct SymbolUpdateStatus
+{
+  std::string new_name;
```

Comment:
As well as explain what each field is supposed to do/mean.

Reply:
Done!